### PR TITLE
Update README to contain proper contents

### PR DESCRIPTION
**Pull request by doosuu**
Updates the README to contain the proper contents.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
...
+
+## Required software
+
+* Visual Studio Code
+* A container runtime supported by Visual Studio Code's devContainer feature. See [here](https://eclipse.dev/velocitas/docs/tutorials/quickstart/container_runtime/)
+
+## Setup steps
+
+1. Clone the repository **or** use Github Codespaces
+2. Open the repository in Visual Studio Code **or** in Github Codespaces
```
Comment (MP91) [inline README.md:13]:
```suggestion
1. Clone the repository and open it in Visual Studio Code **or** use Github Codespaces
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,16 @@
...
+3. Follow the steps through the assisted project creation
+4. When prompted restart the devContainer (:warning:under the hood, we have updated your container configuration:warning:) - if not prompted, manually restart the container using `View` -> `Command Palette...` -> `Dev Containers: Rebuild Container`
```
Comment (MP91) [inline README.md:15]:
```suggestion
4. When prompted, rebuild the devContainer (:warning:under the hood, we have updated your container configuration:warning:) - if not prompted, manually rebuild the container using `View` -> `Command Palette...` -> `Dev Containers: Rebuild Container`
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,16 @@
-# vehicle-app-template
-vehicle-app-template
+# Vehicle App Template
+
+The repository to get you started with Velocitas App development. It contains a very basic setup which will walk you through assisted project creation.
```
Comment (MP91) [inline README.md:3]:
```suggestion
This is a repository to get you started with Velocitas App development. It contains a very basic setup which will walk you through assisted project creation.
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,16 @@
...
+## Required software
+
+* Visual Studio Code
+* A container runtime supported by Visual Studio Code's devContainer feature. See [here](https://eclipse.dev/velocitas/docs/tutorials/quickstart/container_runtime/)
+
+## Setup steps
+
+1. Clone the repository **or** use Github Codespaces
+2. Open the repository in Visual Studio Code **or** in Github Codespaces
+3. Follow the steps through the assisted project creation
```
Comment (MP91) [inline README.md:14]: should we provide some more details about the creation process and what can be selected?

Comment (doosuu) [reply README.md:13]: The steps in the CLI should be assisted enough to be self explanatory. If we describe them here, it will be outdated rather soon.

**Review by MP91 — APPROVED** [review]
lgtm

**Pull request merged**